gulpfile.babel.js

```diff
@@ -156,7 +156,7 @@ function taskScriptsClean() {
 }
 
 function taskScriptsLint() {
-  return gulp.src(`${SCRIPTS_SRC}/app.js`)
+  return gulp.src(`${SCRIPTS_SRC}../**/*.js`)
```

Comment:
Replace this row with this:
`
return gulp.src([`${SCRIPTS_SRC}/**/*.js`, `!${SCRIPTS_SRC}/**/vendor/`, `!${SCRIPTS_SRC}/**/vendor/**/*`])
`